components/bundles/bioformats_package/assembly.xml

```diff
@@ -16,7 +16,6 @@
         <exclude>aopalliance:aopalliance</exclude>
         <exclude>org.aspectj:aspectj*</exclude>
         <exclude>org.slf4j:slf4j-log4j12</exclude>
-        <exclude>log4j:log4j</exclude>
```

Comment:
Does it make sense to keep this exclude with a comment that it explicitly prevents log4j from ever being included through other dependencies?

Reply:
You're correct. That's an exclusion statement not an inclusion and is not even in the right commit. I cannot remember the rationale so I will force push this change away